lib/tasks/support.rake

```diff
@@ -73,12 +73,12 @@ namespace :support do
     elsif !(subscriber.subscriptions.pluck(:subscriber_list_id).include? subscriber_list.id)
       puts "Subscriber #{email_address} does not appear to be signed up for #{subscriber_list_slug}"
     else
-      active_subscriptions = Subscription.active.where(subscriber_list: subscriber_list, subscriber: subscriber)
-      if active_subscriptions.empty?
-        puts "Subscriber #{email_address} already unsubscribed from #{subscriber_list_slug}"
-      else
-        UnsubscribeService.call(subscriber, [active_subscriptions.last], :unsubscribed)
+      active_subscription = Subscription.active.find_by(subscriber_list: subscriber_list, subscriber: subscriber)
+      if active_subscription.present?
```

Comment:
Minor, `present?` is redundant here

Reply:
Yep it is. 

I normally like being explicit in conditionals as I think it reads nicer / comparing against a boolean seems a more traditional approach. I'm not fussed though so have removed.